src/proposer/proposer.cpp

```diff
@@ -220,9 +274,9 @@ std::unique_ptr<Proposer> Proposer::New(
     const Dependency<BlockBuilder> block_builder,
     const Dependency<Logic> proposer_logic) {
   if (settings->node_is_proposer) {
-    return std::unique_ptr<Proposer>(new ProposerImpl(behavior, multi_wallet, network, active_chain, transaction_picker, block_builder, proposer_logic));
+    return std::unique_ptr<Proposer>(new ActiveProposerImpl(behavior, multi_wallet, network, active_chain, transaction_picker, block_builder, proposer_logic));
```

Comment:
NIT: I see `ActiveProposerImpl` receives the same parameters that `PassiveProposerImpl` does + some extra. Would be good to align the order that these extra parameters are on the beginning or at the end for easier reading.